tests/vmi_multus_test.go

```diff
@@ -667,16 +669,27 @@ var _ = Describe("[Serial]SRIOV", func() {
 
 		waitVmi := func(vmi *v1.VirtualMachineInstance) *v1.VirtualMachineInstance {
 			// Need to wait for cloud init to finish and start the agent inside the vmi.
-			vmi, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
```

Comment:
Using `vmi.Namespace` is correct here. The vmi is expected to exist.
If the `Namespace` attribute is not populated, then the wrong `vmi` is used here (a non updated one).